ci/run_ci.py

```diff
@@ -80,6 +80,27 @@ def _run_cpp():
     )
 
 
+def _run_rust():
+    _exec_cmd("rustup component add clippy-preview")
+    _exec_cmd("rustup component add rustfmt")
+    logging.info("Executing fury rust tests")
+    os.chdir(os.path.join(os.getcwd(), "rust"))
```

Comment:
could we read current script path first, then walk from it to rust dir?

Reply:
```
cur_script_abs_path = os.path.split(os.path.realpath(__file__))[0]
os.chdir(os.path.join(cur_script_abs_path, "../rust"))
```

Is this okay?

Reply:
Looks good to me